test/aee.js

```diff
@@ -0,0 +1,47 @@
+require('../aee.js')
+const expect = require('chai').expect
+const sinon = require('sinon')
+const data = require('./testData')
+
+describe('Setting up the environment', () => {
+  it('checks if testing works', () => {
```

Comment:
yeah, it was more about checking my setup, not about theirs, forgot to remove it.